test/src/org/labkey/test/pages/filetransfer/FileTransferConfigPage.java

```diff
@@ -0,0 +1,90 @@
+package org.labkey.test.pages.filetransfer;
+
+import org.labkey.test.Locator;
+import org.labkey.test.pages.LabKeyPage;
+import org.labkey.test.util.LabKeyExpectedConditions;
+import org.openqa.selenium.internal.WrapsDriver;
+
+/**
+ * Created by xingyang on 2/10/17.
+ */
+public class FileTransferConfigPage extends LabKeyPage
+{
+    public static Locator.XPathLocator endpointPathInput = Locator.xpath("//input[@name='endpointPath']");
+    public static Locator.XPathLocator folderInput = Locator.xpath("//input[@name='lookupContainer']");
+    public static Locator.XPathLocator tableInput = Locator.xpath("//input[@name='queryName']");
+    public static Locator.XPathLocator fieldInput = Locator.xpath("//input[@name='columnName']");
```

Comment:
We might consider putting these in an inner Locators class, just to be consistent with other test pages.